Add jupyterhub docker spawner plugin (#1)
RS-DAT/JupyterDockerSpawnerOnSRC: roles/python/README.md

Conda 
=====

> [Conda](https://docs.conda.io) role for the JupyterDask deployment on SURF Research Cloud. Adapted from the [eWaterCycle lab infrastructure](https://github.com/eWaterCycle/infra).

Reviewer: ```suggestion
Python role for JupyterHub with DockerSpawner on [SURF Research Cloud (SRC)](https://servicedesk.surf.nl/wiki/display/WIKI/SURF+Research+Cloud).
```

Requirements
------------

Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.

Role Variables
--------------

```yaml
# Location where conda installer is downloaded to
conda_installer_root: /opt
# Location where conda will be installed
conda_root: /opt/conda
# Name of conda environment to use
conda_environment: jupyterdask
# URL to user-defined environment file (default value: default)
environment_url: default

> ```

Reviewer: ```suggestion
```
